test/form-builder/SelectField.test.tsx

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import { waitFor } from '@testing-library/react';
+
+import SelectField from '../../src/form-builder/SelectField';
+import { buildRenderForm, changeValue } from '../utils';
+
+const renderForm = buildRenderForm({});
+
+const getInput = (container: HTMLElement): HTMLInputElement => {
```

Comment:
:thinking: Can we import the actual type from `web-components` now?

If not, should this be [`HTMLSelectElement`](https://developer.mozilla.org/en-US/docs/Web/API/HTMLSelectElement)?

Reply:
Good call, I'll see if that can be changed.

Reply:
https://github.com/department-of-veterans-affairs/formulate/pull/16/commits/15ea070003722fdf1985a3d44e6cada6345ecefa